app/javascript/react/assets/styles/Typography/Typography.style.tsx

```diff
@@ -0,0 +1,107 @@
+import styled, { css } from "styled-components";
+
+const H1 = styled.h1`
+  font-family: "Roboto";
```

Comment:
I think there is no other font used in the calendar feature, why don't we just set it once in `global-styles.ts`?